### PR TITLE
SEAR-279 Fix for incorrect display of address on facility search list view

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-279

#### Brief feature description
-Before:
    Address was incorrectly displayed on to the UI, when there is any long addresses. 
-After the fix:
   Address is correctly displayed, irrespective of length of the address.
  Also now user have a new functionality of resizing the columns. 

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
-No

#### Code coverage percent diff as a result of this PR?
-No

### CODE REVIEW

**Comment by sunilb0575** [comment]
retest this please

File: app/javascript/search/searchList.js [hunk 1]
```diff
@@ -1,50 +1,101 @@
...
+  {
+    Header: 'Facility Name',
+    minWidth: 70,
+    className: 'facility_name',
+    id: 'facilityName',
+    accessor: d => (<Link to={urlPrefixHelper('/facilities/' + d.id)}>{checkNullOrEmptyValue(d.name)}</Link>)
+  },
+  {
+    Header: 'Facility ID / Approval #',
+    className: 'facility_id',
```
Comment (ratneshraval) [inline app/javascript/search/searchList.js]: Are these `className` necessary?

File: app/javascript/search/searchList.js [hunk 2]
```diff
@@ -1,50 +1,101 @@
...
+    minWidth: 70,
+    accessor: 'license_number'
+  },
+  {
+    Header: 'Facility Type / Facility Source',
+    id: 'facilityTypeSource',
+    className: 'facility_type_source',
+    minWidth: 100,
+    accessor: d => (`${checkForNA(d.type)} / ${checkNullOrEmptyValue(d.facility_source)}`)
```
Comment (Nasir2712) [inline app/javascript/search/searchList.js]: I would rename ```d``` to something more descriptive like data, or props, searchResult

File: app/javascript/search/searchList.js (hunk 1 above)
Comment (ratneshraval) [reply app/javascript/search/searchList.js]: Looks like these are needed for styling and testing.

**Comment by LeonidMarushevskyi** [comment]
jenkins retest this please

**Pull request merged**